JWT Authorization Grant feature to preview (#44805)
keycloak/keycloak: docs/documentation/release_notes/topics/26_5_0.adoc

You can now prevent this by disabling *Include in OpenID Provider Metadata*.

== JWT Authorization Grant new feature

Keycloak 26.5 introduces a new feature called JWT Authorization Grant, which adds support for https://datatracker.ietf.org/doc/html/rfc7523[RFC 7523] to use external signed JWT assertions to request OAuth 2.0 access tokens.

To accept signed JWT assertions, a trust relationship must be established between the external provider and Keycloak. This trust relationship can be configured through an identity provider in a dedicated section of the OpenID Connect v1.0 identity provider, or through the new JWT Authorization Grant identity provider.

> JWT Authorization Grant can be used as alternative to link:{securing_apps_base_link}/token-exchange#_external-token-to-internal-token-exchange[External to internal token exchange V1].

Reviewer: ```suggestion
JWT Authorization Grant is recommended as an alternative to link:{securing_apps_base_link}/token-exchange#_external-token-to-internal-token-exchange[External to internal token exchange V1].
```

I wonder about announcing somehow that this is really preferred instead of external-internal token exchange V1. Could the variant I've suggested work for you?
